db/seeds.rb

```diff
@@ -19,3 +20,10 @@
 #dog_shop items
 pull_toy = dog_shop.items.create(name: "Pull Toy", description: "Great pull toy!", price: 10, image: "http://lovencaretoys.com/image/cache/dog/tug-toy-dog-pull-9010_2-800x800.jpg", inventory: 32)
 dog_bone = dog_shop.items.create(name: "Dog Bone", description: "They'll love it!", price: 21, image: "https://img.chewy.com/is/image/catalog/54226_MAIN._AC_SL1500_V1534449573_.jpg", active?:false, inventory: 21)
+
```

Comment:
Good idea on making seeds!

Reply:
And I could've changed all the names to match the variables (i.e., from 'Kiera Allen' to 'Curtis' and 'Jose')